src/Connection.cpp

```diff
@@ -71,6 +72,7 @@ Connection::~Connection()
 
 void Connection::requestName(const std::string& name)
 {
+    SDBUS_CHECK_SERVICE_NAME(name);
```

Comment:
Extreme nitpick, since all other checks have a empty line after the checks.
```suggestion
    SDBUS_CHECK_SERVICE_NAME(name);
    
```